pkg/asset/templates/kubeconfig.yaml

```diff
@@ -6,10 +6,11 @@ clusters:
     server: {{ .Server }}
```

Comment:
I kinda thought the same, but then I was leaving the .yaml on the template to make editors pick up filetype. I removed it from the rendered copy because it's gross to do `kubectl --kubeconfig=kubeconfig.yaml` everywhere...